test/openshift_version_tests.py

```diff
@@ -17,39 +17,39 @@ class OpenShiftVersionTests(unittest.TestCase):
 
     # Static tests for legacy filters.
     legacy_gte_tests = [{'name': 'oo_version_gte_3_1_or_1_1',
-                         'positive_enterprise_version': '3.2.0',
-                         'negative_enterprise_version': '3.0.0',
+                         'positive_openshift-enterprise_version': '3.2.0',
```

Comment:
This file looks like it may be a culprit of a search and replace based on the name using ``-`` and ``_``: ```negative_openshift-enterprise_version```. Just want to verify ...

Reply:
the search and replace was 's/_enterprise_version/_openshift-enterprise_version/'

Reason being, these dictionary keys are looked up on a "x_{}_version".format(deployment_type) basis.  See line 56.